.integrated_tests.yaml

```diff
@@ -0,0 +1,4 @@
+---
+baselines:
+  bucket: geos-dev
+  baseline: baseline_integratedTests-pr3044-4256-0514e51
```

Comment:
@TotoGaz I guess this is the hash of the last valid reference.

Reply:
Yes I think too. @cssherman ?